bin/tronview

```diff
@@ -14,39 +14,43 @@ from tron.commands.client import TronObjectType
 from tron.commands.cmd_utils import ExitCode
 
 
-def parse_options():
-    usage = "usage: %prog [options] [<job | job run | action | service>]"
-    parser = cmd_utils.build_option_parser(usage)
-    parser.add_option(
-        "--numshown", "-n", type="int", dest="num_displays",
+def parse_cli():
+    parser = cmd_utils.build_option_parser()
+    parser.add_argument(
+        "--numshown", "-n", type=int, dest="num_displays",
         help="Max number of jobs/job-runs shown", default=10,
     )
-    parser.add_option(
+    parser.add_argument(
         "--color", "-c", action="store_true",
         dest="display_color", help="Display in color",
         default=None,
     )
-    parser.add_option(
+    parser.add_argument(
         "--nocolor", action="store_false",
         dest="display_color", help="Display without color",
         default=None,
     )
-    parser.add_option(
+    parser.add_argument(
         "--stdout", "-o", action="count", dest="stdout",
         help="Solely displays stdout", default=0,
     )
-    parser.add_option(
+    parser.add_argument(
         "--stderr", "-e", action="count", dest="stderr",
         help="Solely displays stderr", default=0,
     )
-    parser.add_option(
+    parser.add_argument(
         "--events", action="store_true", dest="show_events",
         help="Show events for the specified entity",
         default=False,
     )
+    parser.add_argument(
+        'name',
```

Comment:
Is the default still OK here and still work? I'm not sure what the default was (None?) but here it is it `[]`? whatever works as long as just running `tronview` still works with nothing extra